router/config.go

```diff
@@ -80,3 +82,13 @@ type RealmConfig struct {
 	// embedding nexus.  A value of nil enables the default filtering.
 	PublishFilterFactory FilterFactory
 }
+
+// TopicEventHistoryConfig stores settings for event history for a topic
+type TopicEventHistoryConfig struct {
+	// Topic URI
+	Topic wamp.URI `json:"topic"`
+	// Matching policy (exact|prefix|wildcard)
```

Comment:
Nitpick, move those two comments to the right hand side of the field. It makes formatting look nicer.

Reply:
👍🏻